packages/pyright-scip/src/treeVisitor.ts

```diff
@@ -574,7 +574,15 @@ export class TreeVisitor extends ParseTreeWalker {
         const existingSymbol = this.rawGetLsifSymbol(decl.node);
         if (existingSymbol) {
             if (decl.node.id === parent.id || decl.node.id === node.id) {
-                this.pushNewOccurrence(node, existingSymbol, scip.SymbolRole.Definition);
+                // Check if the declaration is a function
```

Comment:
```suggestion
                // Check if the declaration is a function
```

The code below is quite clear. 😄